ios/RCTWKWebView/RCTWKWebView.m

```diff
@@ -110,6 +110,19 @@ - (void)setSource:(NSDictionary *)source
   if (![_source isEqualToDictionary:source]) {
     _source = [source copy];
     
+    // Permit loading local files:
+    //    import RNFS from 'react-native-fs';
+    //    <WKWebView source={{ file: RNFS.MainBundlePath + '/data/index.html', allowingReadAccessToURL: RNFS.MainBundlePath }} />
+    NSString *file = [RCTConvert NSString:source[@"file"]];
+    NSString *allowingReadAccessToURL = [RCTConvert NSString:source[@"allowingReadAccessToURL"]];
+    
+    if (file) {
+      NSURL *fileURL = [RCTConvert NSURL:file];
+      NSURL *baseURL = [RCTConvert NSURL:allowingReadAccessToURL];
+      [_webView loadFileURL:fileURL allowingReadAccessToURL:baseURL];
```

Comment:
`loadFileURL` is only available for iOS 9+, which will break the compatibility of iOS 8. You could add a `respondsToSelector` check and log warnings on iOS 8 if users try to load local files.